library/simulink-library/MomentumVelocityControl/src/ComputeReferences.m

```diff
@@ -1,9 +1,12 @@
-function [L_star, s_dot_star_k_1] = ComputeReferences(x_com, x_d_com, Ld, int_L_angular, s_dot_des_k_1, s_des_k, s_k, Gains)
+function [L_star, L_dot_star, s_dot_star_k_1] = ComputeReferences(x_com, x_d_com,  L, L_dot_d, Ld, int_L_angular, s_dot_des_k_1, s_des_k, s_k, totalMass,Gains)
 
-    L_int_lin = Gains.KP_CoM* (x_com- x_d_com); 
-    L_int_ang = Gains.KI_ang* (int_L_angular); 
-    L_int  = [L_int_lin; L_int_ang]; 
-    L_star = Ld - L_int; 
+L_int_lin = Gains.KP_CoM*totalMass*(x_com - x_d_com); 
+L_int_ang = Gains.KI_ang*(int_L_angular); 
+L_int     = [L_int_lin; L_int_ang]; 
+L_star    = Ld-L_int; 
 
-    s_dot_star_k_1 = s_dot_des_k_1 - Gains.KP_Postural*(s_k- s_des_k); 
-end
+L_dot_star = L_dot_d -blkdiag(Gains.KD_CoM, Gains.KP_ang)*(L-Ld) -L_int;
+
+s_dot_star_k_1 = s_dot_des_k_1 - Gains.KP_Postural*(s_k-s_des_k); 
+
+end
```

Comment:
also here missing newline

Reply:
done in 6c3d28fec228d23323ca3c7d07f2342666e69d7e